ideas/310-core-team-proposal.md

```diff
@@ -16,44 +25,34 @@ Desktop is out of scope, as is protocol research. Whitepaper use case implementa
 
 Core needs an accountable, passionate get-shit-done mentality and drive, in contrast to the meandering blame-shifting.
 
+## Additinal contibutors
 
-## Budget
-
-Headcount-based. 8 people.
-
-The swarm also has a discretionary budget for bounty based development using Gitcoin, in order to parallelize development.
-
-Assuming average organizational compensation, this would be roughly 69k/m. Possibly more based on performance, etc.
-A self-imposed ceiling of 100k is proposed for budget bucketing (~10%) for the team as a whole.
-
-Related adjacent activities such as security, devops, marketing, QA, etc will be assumed to be provided and funded by the Gmbh until further notice.
-
-## Core participants
-
-- Adam
 - Andrea MP
-- Chad (PM)
-- DmitryS
-- Igor (tech lead)
+- Dmitry S
 - Andrei M (Design)
-- Pedro
-- Roman
-- Serhy (QA)
+- Serhy/Anna/Nastya(QA)
 
 ## Scope
 
-See also: [team and responsibilities](https://notes.status.im/k0xVSu6nRvS5p0UAWV__BA?view)
-
 Core should focus on problems/features that require multidisciplinary understanding and influence the application architecture. With the highest priority of making Status a usable application for consumption.
 
 For simplicity, the responsibilities are split into 2 categories:
 * **maintenance** - that means this effort is continuous and doesn't have any end date or exit criteria (publishing a mobile release is a good example, we don't stop)
 * **project** - something that has a clear exit criteria, with a few iterations, after which the task is considered completed (for instance, when LES option is there and works)
 
 
-## Process (aka How We Work?)
+## Communications
+(required)
+`status channel (same as swarm id)`:[`#status-core`](http://get.status.im/chat/public/status-core).
```

Comment:
I think one point was brought up re having consistent naming so #310-core, but eh, a bit annoying to change around all the time. Defer to others for now.

Reply:
I'd keep it as is, frankly. The channel is already quite active and I'm not a huge fan of numbers in front of channel names...